js/common/lib/tensor.ts

```diff
@@ -66,7 +66,7 @@ interface TypedTensorBase<T extends Tensor.Type> {
 export declare namespace Tensor {
   interface DataTypeMap {
     float32: Float32Array;
-    uint8: Uint8Array;
+    uint8: Uint8Array | Uint8ClampedArray;
```

Comment:
this may affect type inference. 

we allow using Uint8ClampedArray to create a uint8 tensor; but the underlying data type is always Uint8Array.